src/SFML/Graphics/Font.cpp

```diff
@@ -747,11 +748,11 @@ IntRect Font::findGlyphRect(Page& page, unsigned int width, unsigned int height)
     if (!row)
     {
         int rowHeight = height + height / 10;
-        while ((page.nextRow + rowHeight >= page.texture.getSize().y) || (width >= page.texture.getSize().x))
+        while ((page.nextRow + rowHeight >= pageTextureSize.y) || (width >= pageTextureSize.x))
```

Comment:
The texture size can change inside the loop, so it's wrong to always use the size that was stored at the beginning of the function.
Then the first replacement becomes almost usless, so I would just revert changes to this file completely.

Reply:
Fix it.
https://github.com/SFML/SFML/pull/1732/commits/19b0c3dffc583de96cb3dfedfa82d4d2aa122725

Reply:
In that case, would it make sense to write a comment
```cpp
// Note: texture size can change during the loop
```
, to avoid a future pitfall?